notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -58,7 +58,7 @@
     "%pip uninstall -q -y openvino-dev openvino openvino-nightly\n",
     "%pip install -q --extra-index-url https://download.pytorch.org/whl/cpu\\\n",
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```

Comment:
there are some issues with new commits usage in optimum-intel for llms, I prepared fix, but all responsible for merge people are on holiday, so I'll add workaround, but in separate PR

Reply:
https://github.com/openvinotoolkit/openvino_notebooks/pull/1575

Reply:
shall I update it in this PR ?

Reply:
no, I just let you know, it is global issue that should be fixed in many notebooks, so it is better to keep it separately